packages/hyperion-autologging/src/ALUIEventPublisher.ts

```diff
@@ -289,9 +280,6 @@ export function publish(options: InitOptions): void {
       }
       flowletName += ')';
       let callFlowlet = new flowletManager.flowletCtor(flowletName, ALUIEventGroupPublisher.getGroupRootFlowlet(event));
```

Comment:
Sorry for missing this.

Still it is a bit unclear to me,  so triggerFlowlet is now the only flowlet that is managed?  Call flowlets are turned into trigger flowlets via `setTriggerFlowlet`?

Reply:
This was easy to miss. 
Call flowlets are the vehicle to transfer trigger flowlets around. But the right way to ensure they never accidentially overlap is to push/pop them before/after a sync function, but use a trigger-payload to carry the trigger with async calls. For that logic, you can look at `ALTriggerFlowlet.ts` and `FlowletWrappers.ts` to see how they are moved around. 

The `setTriggerFlowlet` assigns a trigger flowlet to a payload.